protobuf/as/as.proto

```diff
@@ -226,4 +226,10 @@ message HandleTxAckRequest {
 
     // Downlink frame-counter.
     uint32 f_cnt = 2;
+
+    // Gateway ID.
+    bytes gateway_id = 3 [json_name = "gatewayID"];
```

Comment:
There is no need for adding `json_name` as this field is not exposed through the gRPC gateway. It is safe to remove this.